pkg/agent/openflow/client.go

```diff
@@ -618,11 +618,11 @@ func (c *client) GetPodFlowKeys(interfaceName string) []string {
 	return c.getFlowKeysFromCache(c.featurePodConnectivity.podCachedFlows, interfaceName)
 }
 
-func (c *client) InstallServiceGroup(groupID binding.GroupIDType, withSessionAffinity bool, endpoints []proxy.Endpoint) error {
+func (c *client) InstallServiceGroup(groupID, exportedSvcGroupID binding.GroupIDType, withSessionAffinity bool, endpoints []proxy.Endpoint) error {
```

Comment:
What is the expected behavior for a local Service deletion which is used as an Endpoint of the global Service?

Reply:
The local Service will be deleted, then corresponding Endpoint in the global Service will also be deleted as well. When the Endpoints are changed, InstallServiceGroup should be called again without `exportedSvcGroupID`.